src/NinjaTools.FluentMockServer.API/Proxy/Evaluation/EvaluationPipeline.cs

```diff
@@ -0,0 +1,79 @@
+// using AutoMapper;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)